package.json

```diff
@@ -90,10 +98,26 @@
     "should": "13.2.3",
     "should-sinon": "0.0.6",
     "sinon": "^9.0.2",
-    "strip-json-comments": "^3.1.1"
+    "strip-json-comments": "^3.1.1",
+    "ts-node": "^8.10.2",
+    "typescript": "^3.9.3"
   },
   "engines": {
     "node": ">= 12.13.0"
   },
-  "license": "Apache-2.0"
+  "license": "Apache-2.0",
+  "files": [
```

Comment:
You should probably include the following files:
* `.kuzzlerc.sample`
- `*.md`

And I wonder if we shouldn't include those either:
- `Vagrantfile`
- `vagrant/**/*`

Lastly: I cannot recall what's the use for the `config/` and `run/` directories.

Reply:
I'm not sure about Vagrant, those files are very old and I don't even know if it's still working.. 

In `config` Kuzzle try to look for `secrets.enc.json` but I don't know for `run/`

Reply:
I believe it was to host an [unix socket](https://en.wikipedia.org/wiki/Unix_file_types#Socket), used in a past now forgotten when we had our own kuzzle proxy. 
This needs to be verified but if I'm right, it's now unused.